pom.xml

```diff
@@ -18,7 +18,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <cxf.version>3.3.5</cxf.version>
         <jackson.version>2.10.3</jackson.version>
-        <tomcat.version>8.5.51</tomcat.version>
+        <tomcat.version>9.0.33</tomcat.version>
```

Comment:
no one is going to remember to change the tomcat versions when they need to update spring boot versions.

just get rid of this property and use the in built one that the spring bom is providing for you for all tomcat dependencies